pkg/data/data.go

```diff
@@ -0,0 +1,94 @@
+package data
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+)
+
+type Recipient struct {
+	PhoneNumber   string
+	Quantity      string
+	Unit          string
+	Validity      string
+	IsPromoBundle string
+	MetaData      interface{}
+}
+
+/*
+DataRequest represents the structure for the request
+body to be sent to Africa's Talking api endpoint
+*/
+type DataRequest struct {
```

Comment:
Struct documentation is also recommended. It provides developers with intelisense suggestions and also is captured in the [official package docs](https://pkg.go.dev/github.com/edwinwalela/africastalking-go@v0.0.3/pkg/sms#BulkRequest):

Here is how I've documented structs and methods in the other packages: 
https://github.com/EdwinWalela/africastalking-go/blob/0b1a0b87f1315f2b28f5764b745f424486509ded/pkg/sms/sms.go#L22